README.md

```diff
@@ -1,7 +1,17 @@
 # Wikijump
 
-----
+Wikijump is the [SCP Wiki](http://www.scpwiki.com/)'s fork of the unmaintained [Wikidot](https://github.com/gabrys/wikidot).
+It aims to significantly refactor legacy code, remove misfeatures and dead code, and implement new useful features and bugfixes that are needed in mainline Wikidot.
 
-## About
+This project is being developed by the SCP Wiki's [Technical Team](http://05command.wikidot.com/technical-staff-main) as part of Project Foundation.
+Issues are tracked on our [JIRA](https://scuttle.atlassian.net/browse/WDBUGS).
 
-Wikijump is a platform for creating wikis, based on [Wikidot](http://wikidot.com/).
+Questions, comments or suggestions can be suggested in the [Feature Requests forum](http://www.scpwiki.com/forum/c-3335630/feature-requests), or in [`#site11` on SkipIRC](http://www.scpwiki.com/chat-guide).
+
+### Getting Started
+
+We don't have a lot of documentation here yet. See [development.md](docs/development.md) and [install.md](docs/install.md) for information on running a local instance of Wikijump.
+
+### License
```

Comment:
```suggestion
## License
```

Reply:
Why? This is a normal section like the rest of them

Reply:
In one of the other suggested changes I started a 2nd-level "contributing" section, with the 3rd-level "development/getting started" section contained inside it. Licenses aren't contributing so it would need to be bumped up to 2nd-level

Reply:
Oh I see